src/EnergyPlus/ThermalComfort.cc

```diff
@@ -2888,14 +2888,21 @@ namespace ThermalComfort {
             const bool epwFileExists = FileSystem::fileExists(ioFiles.inputWeatherFileName.fileName);
             readStat = 0;
             if (epwFileExists) {
+                // determine number of days in year
+                int DaysInYear;
+                if (DataEnvironment::CurrentYearIsLeapYear) {
+                    DaysInYear = 366;
+                } else {
+                    DaysInYear = 365;
+                }
                 auto epwFile = ioFiles.inputWeatherFileName.open("CalcThermalComfortAdaptiveCEN15251");
-                for (i = 1; i <= 9; ++i) { // Headers
+                for (i = 1; i <= 8; ++i) { // Headers
```

Comment:
Right, it should start taking data on the 9th line.  To make this really clear, you could declare it above:
`int const numHeaderRowsInEPW = 8;`.  Then you don't even need the `// Headers` comment.

Reply:
This I will do shortly. Do you think I should change this bit for the ASH55 above by the way?

Reply:
I have changed the ASH55 as well for now, let me know if you think it inappropriate for this is a CEN15251 PR.